src/tracer/MOM_lateral_boundary_diffusion.F90

```diff
@@ -223,6 +227,8 @@ subroutine lateral_boundary_diffusion(G, GV, US, h, Coef_x, Coef_y, dt, Reg, CS)
         tracer%t(i,j,k) = tracer%t(i,j,k) + (( (uFlx(I-1,j,k)-uFlx(I,j,k)) ) + ( (vFlx(i,J-1,k)-vFlx(i,J,k) ) ))* &
                           G%IareaT(i,j) / ( h(i,j,k) + GV%H_subroundoff )
         if (tracer%id_lbdxy_conc > 0  .or. tracer%id_lbdxy_cont > 0 .or. tracer%id_lbdxy_cont_2d > 0 ) then
+          !### Probably this needs to be multiplied by (h(i,j,k) + GV%H_subroundoff) for consistency
```

Comment:
Yes, this is dimensionally inconsistent. Thanks for catching that. I will send a fix to dev/ncar.

Reply:
In this case, it might make sense for you to route the fix via a PR to dev/gfdl (once PR#1268 is finally resolved) as the most direct path to get these corrections disseminated via the main synchronization fork.

Reply:
The fix has been merged into dev/ncar already (https://github.com/NCAR/MOM6/pull/170). Should I still send a PR to dev/gfdl once this is solved?